lightning/src/chain/keysinterface.rs

```diff
@@ -180,6 +184,7 @@ impl_writeable_tlv_based_enum!(SpendableOutputDescriptor,
 	(0, StaticOutput) => {
 		(0, outpoint, required),
 		(2, output, required),
+		(4, channel_keys_id, required),
```

Comment:
This will fail to read old versions, as they are missing the field. Thus, we need the `channel_keys_id` here to be an `Option` and make the read here `option`al.